tests/test_past_kv.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+import requests
+import torch
+from PIL import Image
+from transformers import AutoProcessor, LlavaForConditionalGeneration
+
+from peft import PrefixTuningConfig, get_peft_model
+
+
+class TestPastKV(unittest.TestCase):
+    def test_past_kv(self):
+        model_id = "trl-internal-testing/tiny-random-LlavaForConditionalGeneration"
+        prompt = "USER: <image>\nWhat are these?\nASSISTANT:"
+        image_file = "http://images.cocodataset.org/val2017/000000039769.jpg"
+
+        # prepare model and inputs
+        model = LlavaForConditionalGeneration.from_pretrained(
+            model_id,
+            torch_dtype=torch.float16,
+            low_cpu_mem_usage=True,
+        )
+        processor = AutoProcessor.from_pretrained(model_id)
+        raw_image = Image.open(requests.get(image_file, stream=True).raw)
```

Comment:
I think we can simplify this by just creating a random image, as the content does not matter for this test:

```python
raw_image = np.random.randint(0, 255, (224, 224, 3), dtype=np.uint8)
```